### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by trouta23**
Update README to reflect the new `location` method

### CODE REVIEW

**Comment by trouta23** [comment]
You'll see a lot of whitespace changes, but the `location` method change is in there. I also didn't touch on doing something like `TelephoneNumber.parse('+8222123456').location(:ja) => 'ソウル特別市'` to keep it in scope of phone_object

File: README.md [hunk 1]
```diff
@@ -79,74 +78,83 @@ After that you have the following instance methods available to you.
...
    #<TelephoneNumber::Country:0x007fb976267410
    @country_code="1",
    @country_id="US",
    ...
   ```
-  
+
+- ### `location(:en)`
+
+  Returns the geolocation of the number.
```
Comment (akfernun) [inline README.md]: Should we just say "location" as opposed to "geolocation"?

**Comment by akfernun** [comment]
@trouta23, can we add in the locale functionality? It doesn't need to be anything long, but just one or two sentences about how users can pass in a locale to the `location` method, and it will respond appropriately?

File: README.md (hunk 1 above)
Comment (trouta23) [reply README.md]: Yea. I went back and forth on it trying to decide if it was better to tie it to the method name or reference the file names.

**Comment by trouta23** [comment]
Yea I'll throw in the locale example.

**Pull request merged**